SLHCUpgradeSimulations/Configuration/python/phase1TkCustoms.py

```diff
@@ -273,14 +270,15 @@ def customise_harvesting(process):
     process.DQMOffline_SecondStep_PrePOG.remove(process.normalizePlotsRealMuonsData)
     process.postValidation.remove(process.runTauEff)
 
+    # TODO this should be removed by the era switch, but I can't find where it comes from
+    process.dqmHarvesting.remove(process.sipixelCertification)
+

```

Comment:
I will look into this one again once your changes are merged. The problem is that for the new Phase1 DQM, I removed the old Harvesting, which was still in place (because it does not crash), but it is useless and also it will become partially obsolete with the new DQM (Summaries are handled per-plugin).

With the old Harvesting replaced/removed, the Certification crashes, since it relied on the `MonitorElements` being present, I guess.